### DISCUSSION TITLE
选择服务器节点什么时候上线

### DISCUSSION THREAD

**Discussion by lsq88334753** [getlantern/lantern-forum-cn#278] [General]
总是自动接到HK节点，gpt和youtube都用不了，体验非常差

**Comment by MaxMKMax** [comment]
@lsq88334753 给您带来的不便，敬请谅解。了解您的痛点。是的，团队实际上正在做这件事，仍然需要更多时间来发布此功能。抱歉等待。谢谢。

**Reply by mehuba** [reply]
非常需要这个功能，请尽快开发！
